build.gradle

```diff
@@ -0,0 +1,98 @@
+plugins {
+    id 'java-gradle-plugin'
+    id 'groovy'
+    id "com.gradle.plugin-publish" version "0.10.0"
+    id "com.github.hierynomus.license" version "0.14.0"
+}
+
+group = 'org.moditect'
+version = '1.0.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+    jcenter()
+    gradlePluginPortal()
+    flatDir {
+        dirs "${rootProject.projectDir}/lib"
+    }
+}
+
+sourceCompatibility = 1.8
+
+def defaultEncoding = 'UTF-8'
+[compileJava, compileTestJava]*.options*.encoding = defaultEncoding
+
+[compileGroovy, compileTestGroovy]*.options*.encoding = defaultEncoding
+[compileGroovy, compileTestGroovy]*.groovyOptions*.encoding = defaultEncoding
+
+sourceSets.main.java.srcDirs = []
+sourceSets.main.groovy.srcDirs = ["src/main/java", "src/main/groovy"]
+
+configurations {
+    compile {transitive = true}
```

Comment:
Transitive dependencies should be resolved by default, aren't they? Why is this needed here?